daemon/events.go

```diff
@@ -80,6 +80,29 @@ func (daemon *Daemon) LogNetworkEventWithAttributes(nw libnetwork.Network, actio
 	daemon.EventsService.Log(action, events.NetworkEventType, actor)
 }
 
+// LogDaemonEventWithAttributes generates an event related to the daemon itself with specific given attributes.
+func (daemon *Daemon) LogDaemonEventWithAttributes(action string, attributes map[string]string) {
+	if daemon.EventsService != nil {
+		for _, label := range daemon.configStore.Labels {
```

Comment:
Thanks for the review @mlaventure. It seems like `daemon.configStore.Labels` is defined as `[]string`. Not sure if I could avoid the parsing?